### DISCUSSION TITLE
Collaborative Editing

### DISCUSSION THREAD

**Discussion by ZehuaZhang** [facebook/lexical#5500] [Ideas]
Really awesome editor! Is there a plan to implement collaborative editing?

If i want to implement my own. Do you have any recommendations, with MIT license? And brief steps of how to achieve with lexical editor?

**Comment by marekzan** [comment]
Maybe this might help you: https://lexical.dev/docs/collaboration/react

**Reply by ZehuaZhang** [reply]
I heard about Yjs before, time to looked into it, thank you!
